src/bun.js/test/expect.zig

```diff
@@ -1774,6 +1775,7 @@ pub const Expect = struct {
             globalObject.throw(comptime Output.prettyFmt(fmt, true), .{ expected_fmt, value_fmt });
             return .zero;
         }
+	globalObject.throw(comptime Output.prettyFmt(fmt, false), .{ expected_fmt, value_fmt });
```

Comment:
Please make sure to run the Zig formatter. If you use VSCode and our recommended extension it should do it for you.

Also, could we add some tests in `test/js/bun/test` to make sure this doesn't regress?

Reply:
Actually, I think we can pass on this test for now. it's a bit silly to test for this specifically. later we should add overall tests to check output with ansi, ansi + color, color, and none; but that's out of scope for this pr.

Reply:
Thanks for the zig formatter reminder, and sorry for not knowing about it.

I agree about the tests, FWIW. Adding a NO_COLOR option to the test matrix would also work.